airflow/www/static/js/task_instances.js

```diff
@@ -98,6 +98,7 @@ export default function tiTooltip(ti, { includeTryNumber = false } = {}) {
   if (ti.operator !== undefined) {
     tt += `Operator: ${escapeHtml(ti.operator)}<br>`;
   }
+  tt += `Trigger Rule: ${task.trigger_rule}<br>`;
```

Comment:
Wrap this in an if statement checking that task and trigger_rule are defined.

Reply:
An if statement was added in commit 3412688